### PR TITLE
Improve the warning message for analyzer package - Microsoft.CodeAnal…

### PR DESCRIPTION
**Pull request by mavasani**
…ysis version mismatch

### CODE REVIEW

**Comment by mavasani** [comment]
@jinujoseph @kuhlenh @333fred @heejaechang - can you review?

@mattscheffer, does the new message seem more clear?

File: src/Analyzer.Utilities/AnalyzerUtilitiesResources.resx [hunk 1]
```diff
@@ -151,10 +151,10 @@
     <value>Security</value>
   </data>
   <data name="VersionCheckDescription" xml:space="preserve">
-    <value>Analyzers in this package are preview version and are tied to a specific API version of Microsoft.CodeAnalysis. You have a mistatch between the analyzer and Microsoft.CodeAnalysis version and should either switch your analyzer NuGet package/VSIX to a matching version of the analyzer package or move to a  different version of Visual Studio.</value>
+    <value>Analyzers in this package are preview version and are tied to a specific API version of Microsoft.CodeAnalysis. You have a mistatch between the analyzer and Microsoft.CodeAnalysis version and should switch your analyzer NuGet package/VSIX to a matching version of the Microsoft.CodeAnalysis.</value>
```
Comment (jinujoseph) [inline src/Analyzer.Utilities/AnalyzerUtilitiesResources.resx]: 'mismatch' spelling

Comment (mavasani) [inline src/Analyzer.Utilities/AnalyzerUtilitiesResources.resx]: Oh yes :)

**Comment by mattscheffer** [comment]
The new message looks good to me.

**Comment by jinujoseph** [comment]
cc @kuhlenh

**Comment by kuhlenh** [comment]
LGTM

**Pull request merged**